xrdpvr/xrdpvr.c

```diff
@@ -24,6 +24,27 @@
 #include "xrdpvr.h"
 #include "xrdpvr_internal.h"
 
+/* Support newer versions of ffmpeg and libav */
```

Comment:
That's not specific enough. Please specify versions. "Newer" today could be "stone age" 5 years from now.